src/core/api/payment_transactions_gateway_api/internal/v1/_openapi.json.tpl

```diff
@@ -1604,6 +1624,30 @@
           "error"
         ]
       },
+      "XPayRefundResponse409": {
+        "type": "object",
+        "properties": {
+          "requestId": {
+            "type": "string",
+            "example": "77e1c83b-7bb0-437b-bc50-a7a58e5660ac"
+          },
+          "error": {
+            "type": "string",
+            "example": "Payment in DENIED status"
+          },
+          "status": {
+            "type": "string",
+            "enum": [
+              "DENIED"
```

Comment:
Same as below

Reply:
Yes I agree with you, but these specs came from [pgs repo](https://github.com/pagopa/pagopa-payment-transactions-gateway)